internal/cli/cmd/version.go

```diff
@@ -44,8 +46,9 @@ func newCmdVersion() *cobra.Command {
 			case GitHash != "":
 				gitInfo = fmt.Sprintf("@%s", GitHash)
 			}
-			// TODO: add support for reporting the Cilium version
-			fmt.Printf("cilium-cli: v%s%s compiled with %v on %v/%v\n", Version, gitInfo, runtime.Version(), runtime.GOOS, runtime.GOARCH)
+			// TODO: add support for reporting the Cilium version running in
+			// the cluster, if any.
+			fmt.Printf("cilium-cli: v%s%s (%s images) compiled with %v on %v/%v\n", Version, gitInfo, defaults.CiliumVersion(), runtime.Version(), runtime.GOOS, runtime.GOARCH)
```

Comment:
Given that we also want to report the Cilium version running in the cluster (if any) in the future and to clearly separate the different versions, I'd probably add a separate line reporting this, e.g.

```suggestion
			fmt.Printf("cilium-cli: v%s%s compiled with %v on %v/%v\n", Version, gitInfo, runtime.Version(), runtime.GOOS, runtime.GOARCH)
			fmt.Printf("images: %s\n", defaults.CiliumVersion())
```

And then for the Cilium version running in the cluster, we'd probably add another line. This would also approximately correspond to what the in-cluster `cilium` CLI reports and it would keep the version lines somewhat consistent across tools.